oneview/resource_logical_interconnect_group.go

```diff
@@ -876,15 +876,15 @@ func resourceLogicalInterconnectGroupRead(d *schema.ResourceData, meta interface
 		})
 	}
 
-	interconnectMapEntryTemplateCount := d.Get("interconnect_map_entry_template.#").(int)
-	for i := 0; i < interconnectMapEntryTemplateCount; i++ {
-		currBayNum := d.Get("interconnect_map_entry_template." + strconv.Itoa(i) + ".bay_number")
-		for j := 0; j < len(logicalInterconnectGroup.InterconnectMapTemplate.InterconnectMapEntryTemplates); j++ {
-			if currBayNum == interconnectMapEntryTemplates[j]["bay_number"] {
-				interconnectMapEntryTemplates[i], interconnectMapEntryTemplates[j] = interconnectMapEntryTemplates[j], interconnectMapEntryTemplates[i]
-			}
-		}
-	}
+	// interconnectMapEntryTemplateCount := d.Get("interconnect_map_entry_template.#").(int)
+	// for i := 0; i < interconnectMapEntryTemplateCount; i++ {
+	// 	currBayNum := d.Get("interconnect_map_entry_template." + strconv.Itoa(i) + ".bay_number")
+	// 	for j := 0; j < len(logicalInterconnectGroup.InterconnectMapTemplate.InterconnectMapEntryTemplates); j++ {
+	// 		if currBayNum == interconnectMapEntryTemplates[j]["bay_number"] {
+	// 			interconnectMapEntryTemplates[i], interconnectMapEntryTemplates[j] = interconnectMapEntryTemplates[j], interconnectMapEntryTemplates[i]
+	// 		}
+	// 	}
+	// }
```

Comment:
This can be removed as it is no longer needed.

Reply:
Removed.